code/__DEFINES/blob_defines.dm

```diff
@@ -55,7 +55,7 @@
 #define BLOB_RESOURCE_HP_REGEN                      15
 #define BLOB_RESOURCE_MIN_DISTANCE                  3           // Minimum distance between resource blobs
 #define BLOB_RESOURCE_GATHER_DELAY                  4 SECONDS   // Gather points when pulsed outside this interval
-#define BLOB_RESOURCE_GATHER_ADDED_DELAY            0.4 SECONDS// Every additional resource blob adds this amount to the gather delay
+#define BLOB_RESOURCE_GATHER_ADDED_DELAY            0.25 SECONDS// Every additional resource blob adds this amount to the gather delay
```

Comment:
One of us is misunderstanding this.
My understanding from the comment is that every 4 seconds, resource nodes gather points once pulsed. For every resource blob, it increases from 4 seconds to 4.4, 4.8, 5.2 seconds and so on, and this would make it 4 seconds, 4.25, 4.5, 4.75 seconds and so on.
So it's pulse more often, gather resources more often.

Reply:
Yeah. That is correct. That was the old delay.
Its supposed to be a buff. It was changed in the old PR.
When I made the cost of the tiles 3-4, I figured this could switch back.
" Buffs resource blob scaling back to the normal amount, because it isn't going to be able to place as many of them."
I mean, I can keep it if you want. But I am not sure about the misunderstanding.